app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -80,6 +80,9 @@
         public Callback.a1<Spannable> highlighter;
         public String extraFilter = null;
 
+        public String neutralButtonText = null;
```

Comment:
can be int then as well like the others, pass **R.string.search_and_replace**.

I will push that to branch self